### PR TITLE
re-enable LEGO.xml

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: src/chrome/content/rules/LEGO.xml [hunk 1]
```diff
@@ -1,141 +1,62 @@
...
-	<securecookie host="^(?:aboutus|account2?|services\.account|assets|rebrick)\.lego\.com$" name=".+" />
-
+	<target host="lc-www-live-s.legocdn.com" />
+	<test url="http://lc-www-live-s.legocdn.com/r/www/r/globalnavigationservices/content/images/lego-logo-menu.svg" />
 
-	<!--rule from="^http://www\.lego\.com/"
-		to="https://wwwsecure.lego.com/" /-->
+	<!-- 403 -->
+	<exclusion pattern="^http://jobsearch\.lego\.com/$" />
+	<test url="http://jobsearch.lego.com/sap/bc/webdynpro/sap/" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/LEGO.xml]: Can we put this exclusion with the target? I literally had to delete my previous review when I saw it.

Comment (J0WI) [reply src/chrome/content/rules/LEGO.xml]: Do you mean the sorting?

Comment (gloomy-ghost) [reply src/chrome/content/rules/LEGO.xml]: yes

Comment (J0WI) [reply src/chrome/content/rules/LEGO.xml]: Sure, done.

**Comment by gloomy-ghost** [comment]
> ERROR src/chrome/content/rules/LEGO.xml: Fetch error: http://services.community.lego.com/ => https://services.community.lego.com/: (6, 'Could not resolve host: services.community.lego.com')

**Comment by J0WI** [comment]
Thanks, I removed it from the list.

**Comment by gloomy-ghost** [comment]
merged, thanks!

**Pull request merged**